README.md

```diff
@@ -110,18 +112,46 @@ For the first version of the package, add this line to the beginning of `preInst
 |-----------------|---|
 |2.0|not supported|
 |3.0|optional|
+|4.0|optional|
 
 Introduced in `packagingVersion` `3.0`, `.minDcosReleaseVersion` can be specified as a property of `package.json`.
 When `.minDcosReleaseVersion` is specified the package will only be made available to DC/OS clusters with a DC/OS
 Release Version greater than or equal to (`>=`) the value specified.
 
 For example, `"minDcosReleaseVersion" : "1.8"` will prevent the package from being installed on clusters older than DC/OS 1.8.
 
+###### `.upgradesFrom`
+|Packaging Version|   |
+|-----------------|---|
+|2.0|not supported|
+|3.0|not supported|
+|4.0|optional|
+
+Introduced in `packagingVersion` `4.0`, `.upgradesFrom` can be specified as a property of `package.json`.
+When `.upgradesFrom` is specified this indicates to users that the package is able to upgrade from any of
+the versions listed in the property. It is the resposibility of the package creator to make sure that this
+is indeed the case. If this property is set then `minDcosReleaseVersion` must be set to `1.10`
```

Comment:
its confusing. If upgrades will be available starting 1.10, we should probably add it to the documentation at that point.